syft/workers/websocket_client.py

```diff
@@ -15,6 +15,9 @@
 logger = logging.getLogger(__name__)
 
 
+TIMEOUT_INTERVAL = 9999999
```

Comment:
what is the default for this if nothing is passed (also could you add a justification for why this was chosen)

Reply:
This value was just picked to avoid sockets from timing out. 9999999 seconds looked like a sufficiently large amount of time to me. If nothing is passed or no timeout is set, then you can refer to the default values listed in the respective documentation for sockets:

https://docs.python.org/3/library/socket.html
If no timeout is supplied, the global default timeout setting returned by getdefaulttimeout() is used.

And for websockets:

https://websockets.readthedocs.io/en/stable/api.html

ping_timeout=20, close_timeout=10 are the default values.